Cargo.toml

```diff
@@ -9,7 +9,7 @@ edition = "2021"
 [profile.release]
 debug = false
 lto = true
-opt-level = 'z'
+opt-level = 's'
```

Comment:
Starting to run into the (uncompressed) canister size limit. This was sufficient to fix the issue for the time being.